cmd/migrator/app/migrator.go

```diff
@@ -24,6 +24,8 @@ const (
 
 var (
 	kubeconfigPath = flag.String("kubeconfig", "", "absolute path to the kubeconfig file specifying the apiserver instance. If unspecified, fallback to in-cluster configuration")
+	kubeAPIQPS     = flag.Float64("kube-api-qps", float64(rest.DefaultQPS), "QPS to use while talking with kubernetes apiserver.")
```

Comment:
in kube-controller-manager these are both 32 bit precision.  You even cast down below.

Reply:
Thanks. I will take a look

Reply:
updated to follow kube-controller-manager style flag initialization